packages/core-components/src/layout/Sidebar/Page.tsx

```diff
@@ -39,6 +39,7 @@ const useStyles = makeStyles<Theme, StyleProps>(
   theme => ({
     root: {
       width: '100%',
+      height: '100vh',
```

Comment:
This seems super risky, afaik it doesn't work that well either. Any other options for this? 😅

Reply:
Managed to fix that in a different way, please take a look.